src/main/java/cz/metacentrum/perun/googlegroupconnector/GoogleGroupsServiceImpl.java

```diff
@@ -800,4 +826,286 @@ private void deleteMember(String groupName, String memberId) throws GoogleGroups
 		}
 	}
 
+
+	/**
+	 * New class representing tuple of team drive and its users from CSV file.
+	 */
+	public class DriveWithMembers {
+		private TeamDrive teamDrive;
+		private List<User> driveUsers = new ArrayList<>();
+
+		DriveWithMembers(TeamDrive teamDrive, List<User> driveUsers) {
+			this.teamDrive = teamDrive;
+			this.driveUsers = driveUsers;
+		}
+
+		public TeamDrive getTeamDrive() {
+			return teamDrive;
+		}
+
+		public void setTeamDrive(TeamDrive teamDrive) {
+			this.teamDrive = teamDrive;
+		}
+
+		public List<User> getDriveUsers() {
+			return driveUsers;
+		}
+
+		public void setDriveUsers(List<User> driveUsers) {
+			this.driveUsers = driveUsers;
+		}
+	}
+
+	@Override
+	public List<DriveWithMembers> parseTeamDrivesFile(File teamDriveFile) {
+
+		List<DriveWithMembers> result = new ArrayList<>();
+		FileReader fileReader = null;
+
+		try {
+
+			fileReader = new FileReader(teamDriveFile);
+
+			char separator = ';';
+			CSVReader reader = new CSVReader(fileReader, separator);
+
+			List<String[]> lines = reader.readAll();
+			if (lines != null && !lines.isEmpty()) {
+
+				for (String[] line : lines) {
+
+					if (line.length < 2) {
+						log.error("TeamDrive file contains row with less than 2 columns: {}", line);
+						throw new IllegalArgumentException("TeamDrive file contains row with less than 3 columns:" + line[0]);
+					}
+
+					TeamDrive teamDriveResult = new TeamDrive();
+					List<User> userListResult = new ArrayList<>();
+
+					teamDriveResult.setName(line[0].toLowerCase());
```

Comment:
Why do you call ``lowerCase()`` on team drive name ? Is there a constraint on Google side ?